packages/xo-web/src/common/xo/index.js

```diff
@@ -2250,6 +2250,24 @@ export const toggleSrMaintenanceMode = sr => {
   })
 }
 
+export const reclaimSrSpace = async sr => {
+  await confirm({
+    icon: 'sr-reclaim-space',
+    title: _('srReclaimSpace'),
+    body: _('srReclaimSpaceConfirm'),
+  })
+
+  try {
+    await _call('sr.reclaimSpace', { id: resolveId(sr) })
+    success(_('srReclaimSpace'))
+  } catch (err) {
+    if (err?.data?.message?.includes('Operation not supported')) {
+      throw new Error('Space reclaim not supported. Only supported on block based/LVM based SRs.')
```

Comment:
Log the original error, it may contains useful info.

Reply:
Not addressed

Reply:
I logged it in the server. You meant logging it in the browser? At the moment, I'm not sending the whole XML to client.

Reply:
Ok that's good enough :slightly_smiling_face: